src/main/resources/movies.cql

```diff
@@ -0,0 +1,19 @@
+CREATE (:Actor{name:'Keanu Reeves'})-[:ACTED_IN]->(:Movie{name:'Matrix'})-[:IN_GENRE]->(:Genre{name:'Action'});
```

Comment:
Please use .cypher extension for cypher scripts

Reply:
Done